das_client/lib/app/pages/journey/train_journey/widgets/train_journey.dart

```diff
@@ -104,9 +108,14 @@ class TrainJourney extends StatelessWidget {
   }
 
   List<BaseRowBuilder> _rows(BuildContext context, Journey journey, TrainJourneySettings settings) {
-    final rows = journey.data.groupBaliseAndLeveLCrossings(settings.expandedGroups);
+    final currentTrainSeries = settings.selectedBreakSeries?.trainSeries ?? journey.metadata.breakSeries?.trainSeries;
```

Comment:
Ich würde diese beiden Zeilen direkt in die Methode `_isCurvePointWithoutSpeed` nehmen. Also journey und settings übergeben.